### DISCUSSION TITLE
Show size of tumblr blog before downloading

### DISCUSSION THREAD

**Discussion by wirescarryingme** [TumblThreeApp/TumblThree#272] [Ideas]
I want to be able to see the size of the tumblr blog I am downloading before I download it so I know if I have enough space to download the blog.

**Comment by hrxn** [comment]
Would be nice, but I'm not sure if that is even possible..

What could be done instead, is to show simply the total postcount. Given with the post type, a rough approximation could be possible.

**Comment by wirescarryingme** [comment]
That is a good point that it might not be possible, so that is a good alternative to my suggestion!

**Comment by thomas694** [comment]
The size in bytes cannot be shown, as that information isn't available.

For normal blogs the number of posts is shown in the details pane and in the blog list column "number of downloadable items" when adding the blog. More detailed numbers based on post type are only available after a crawl process has finished (or numbers up until then if interrupted). With these numbers you can do a rough estimation.
For blogs of type "hidden" the number of posts cannot be shown when adding the blog.
